config/test.exs

```diff
@@ -13,13 +13,11 @@ config :franklin, Franklin.Repo,
   pool: Ecto.Adapters.SQL.Sandbox,
   pool_size: 10
 
-config :franklin, Franklin.EventStore,
-  serializer: Commanded.Serialization.JsonSerializer,
-  username: "postgres",
-  password: "postgres",
-  database: "franklin_event_store_test",
-  hostname: "localhost",
-  pool_size: 10
+config :franklin, Franklin.CommandedApplication,
+  event_store: [
+    adapter: Commanded.EventStore.Adapters.InMemory,
+    serializer: Commanded.Serialization.JsonSerializer
+  ]
```

Comment:
This change was made to get testing to work. It sets up an in-memory EventStore and thus removes the need for a dedicated test EventStore database.

This does, I think, prevent us from doing async tests. I am asking about this here: <https://github.com/commanded/commanded/discussions/499>

We may revert this to a database if we can somehow set up the event store to use the same sandbox-like setup we have for the main database, but I'm not sure yet.